### ISSUE TITLE
[Offline] What happens when `storage.store` is called twice with the same `manifestURI` as input ?

### ISSUE THREAD

**Issue by loicraux** [shaka-project/shaka-player#6338]
**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
Yes

**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**If the question is related to FairPlay, have you read the tutorial?**
N/A

**What version of Shaka Player are you using?**
Latest version 4.7.11

**What browser and OS are you using?**
Chrome browser
Windows, MacOS & Linux systems

**Please ask your question**
What happens if the `storage.store` API is called twice _with the same `manifestUri` input parameter_ ?
For instance like this :
```ts
const storedContent1 = await storage.store(manifestURI).promise;
const storedContent2 = await storage.store(manifestURI).promise;
console.log(storedContent1.offlineUri === storedContent2.offlineUri); // Is this true or false ?
```

- Is the content downloaded twice and we get two different offlineURIs ?
- Is the content downloaded only once but we get two different offlineURIs ?
- Is the content downloaded only once and we get the same offlineURIs ?

Thanks in advance for your answer !

**Comment by joeyparrish** [comment]
Looks like a short enough code snippet to just run it and find out.  😄 

I believe this would download the content twice, and give you two different offline URIs.  But you can try it and confirm if you like.

**Comment by loicraux** [comment]
Yes, I admit I was a bit lazy on that one... 🤣  I figured that someone who worked on the offline storage code could have given me a quick straight answer...

After inspecting the shaka player code, I can confirm that if you download the same `manifestUri` twice using the `storage.store` API, the content is downloaded twice and you get in return two different `offlineUri`, which seems to me to be the right behavior...

Do you think this should be added to the documentation ?

**Comment by joeyparrish** [comment]
I would be happy to review docs PRs.

The way I tend to think about this is, "if I couldn't tell, maybe we should add a note somewhere".  It's similar to me to the other thing I try to remember, which is, "if my code reviewer had to ask how this works, I should add a code comment".

So, yes, please add a note to the docs that would have saved you the trouble of filing this issue!

**Comment by loicraux** [comment]
Do I have to file a new issue of type "bug" or is this one (of type "question") is OK for a PR ?

**Comment by joeyparrish** [comment]
No, we don't have to have a bug for every PR.  But you should tag this issue in the PR, with either "Closes #6338" to close this question when the PR lands, or "Issue #6338" to reference it without closing it.

We are also happy to relabel this as a docs issue... which I'm going to do now.
